developer/src/server/package.json

```diff
@@ -24,13 +22,13 @@
     "node-windows-trayicon": "keymanapp/node-windows-trayicon#keyman-16.0"
   },
   "devDependencies": {
-    "@keymanapp/resources-gosh": "*",
     "@keymanapp/keyman-version": "*",
+    "@keymanapp/resources-gosh": "*",
     "@types/chai": "^4.3.0",
     "@types/express": "^4.17.13",
     "@types/mocha": "^9.1.0",
     "@types/multer": "^1.4.7",
-    "@types/node": "^17.0.0",
+    "@types/node": "^20.4.1",
```

Comment:
Are you on the current Node.js? (newer than 18.18 LTS)

Reply:
Running on 18.x; the types needed to be brought up to 18.x minimum and most types refs were 20.4.1 so I just went with that. It's only used for typescript engine within vscode, so it's not critical.